src/api/app/assets/stylesheets/webui/sidebar-collapsible.scss

```diff
@@ -0,0 +1,65 @@
+#left-navigation-area {
+  transition: width 0.3s;
+  width: $left-navigation-width;
+
+  .section-name {
+    white-space: nowrap;
+  }
+
+  .section-delimiter {
+    visibility: hidden;
+    border-top: 1px solid $gray-500;
+  }
+
+  #toggle-sidebar-button {
+    width: $left-navigation-width;
+    transition: width 0.3s;
+    position: fixed;
+    bottom: 0;
+  }
+
+  #left-navigation {
+    position: sticky;
+    top: $top-navigation-height;
+    left: 0;
+    max-height: calc(100vh - #{$top-navigation-height});
+    overflow-y: auto;
+    overflow-x: hidden;
+  }
+  & a {
+    color: $gray-300;
+    &:hover { color: white; }
+  }
+  & i {
+    width: 1.2rem;
+  }
+}
+
+#left-navigation-area.collapsed {
+  width: $left-navigation-collapsed-width;
+
+  .nav-item-name, .collapse-button-name {
+    display: none;
+  }
+
+  .section-name {
+    visibility: hidden;
+  }
+
+  #toggle-sidebar-button {
+    width: $left-navigation-collapsed-width;
+  }
+  .section-delimiter {
+    visibility: visible;
+  }
+}
+
+// This prevents content from breaking the layout by overflowing outside the viewport
+#content {
+  // 2rem is the sum of the left and right padding for the content
+  max-width: calc(100vw - (#{$left-navigation-width} + 2rem));
```

Comment:
As we discussed together, this seems to be the line which is causing issues right now in the feature specs.

Reply:
Yep, fixed that. Thank you.